presentation/src/main/java/com/cointrend/presentation/ui/coindetail/CoinDetailScreen.kt

```diff
@@ -66,17 +66,6 @@ fun CoinDetailScreen(
                     }
                 },
                 actions = {
-                    /*
-                    IconButton(onClick = {}) {
-                        Icon(
-                            imageVector = Icons.Filled.Notifications,
-                            contentDescription = "Localized description",
-                            tint = Favourite
-                        )
-                    }
-
-                     */
-
```

Comment:
There is still this change in CoinDetailScreen. Please, revert it.

Reply:
it's good we have applied the changes